index.html

```diff
@@ -4,25 +4,31 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Portfolio</title>
+  <title>Uchechi Nwaka's Portfolio</title>
   <link rel="stylesheet" href="style.css">
 </head>
 <body>
   <header class="header">
     <nav class="Toolbar-container">
-      <a href="#">Portfolio</a>
+      <a id="NW" href="#">Uchechi Nwaka</a>
       <img src="images/menu.png" alt="menu">
+      <div class="Right">
+        <a href="#" id="Portfolio">Portfolio</a>
+        <a href="#">About</a>
+        <a href="#" id="Contact">Contact</a>
+      </div>
     </nav>
     <main class="container">
-      <h1>
-        I'm Uchechi<br>
-        Glad to see<br>
-        you!
-      </h1>
-        <p>
-        I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
-        </p>
-        <h3>LET'S CONNECT</h3>
+      <div class="Text">
+        <h1>
+          I'm Uchechi<br>
+          Glad to see<br>
+          you!
+        </h1>
+          <p>
+          I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+          </p>
+          <h3>LET'S CONNECT</h3>
```

Comment:
- [ optional] I'm so sorry that this was not mentioned before now, kindly make this element either a link `a `tag or `button `tag. It's suppose to be a clickable event. 👍